presto-main/src/main/java/com/facebook/presto/memory/LocalMemoryManager.java

```diff
@@ -49,6 +49,13 @@ public LocalMemoryManager(NodeMemoryConfig config)
         configureMemoryPools(config, Runtime.getRuntime().maxMemory());
```

Comment:
What do you think about using a telescoping constructor since one is a prefix of the other?
```
    @Inject
    public LocalMemoryManager(NodeMemoryConfig config)
    {
       this(config, Runtime.getRuntime().maxMemory());
    }
```